src/audio/mixin_mixout/mixin_mixout.c

```diff
@@ -626,6 +600,70 @@ static int mixout_reset(struct processing_module *mod)
 	return 0;
 }
 
+static void base_module_cfg_to_stream_params(const struct ipc4_base_module_cfg *base_cfg,
+					     struct sof_ipc_stream_params *params);
+
+/* params are derived from base config for ipc4 path */
+static int mixin_params(struct processing_module *mod)
+{
+	struct sof_ipc_stream_params *params = mod->stream_params;
+	struct mixin_data *md = module_get_private_data(mod);
+	struct comp_dev *dev = mod->dev;
+	struct list_item *blist;
+	int ret;
+
+	comp_dbg(dev, "mixin_params()");
+
+	base_module_cfg_to_stream_params(&mod->priv.cfg.base_cfg, params);
+
+	/* Buffers between mixins and mixouts are not used (mixin writes data directly to mixout
+	 * sink). But, anyway, let's setup these buffers properly just in case.
```

Comment:
This feels quit out-of-place, but as this is existing comment (and code) moved as-is, not need to block because of this.

Reply:
the comment could be a bit "kinder" though at least, stating some "better sounding" reason than "just in case," e.g. "for correctness," "for consistency" etc. But yes, it's just a verbatim copy of the current comment